elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/scaladsl/ElasticsearchFlow.scala

```diff
@@ -141,12 +126,55 @@ object ElasticsearchFlow {
     require(settings.retryLogic == RetryNever,
             "`withContext` may not be used with retrying enabled, as it disturbs element order")
     Flow[WriteMessage[T, C]]
-      .via(createWithPassThrough(indexName, typeName, settings, writer))
+      .batch(settings.bufferSize, immutable.Seq(_)) { case (seq, wm) => seq :+ wm }
+      .via(
```

Comment:
IUC you can still `.via(stageFlow)` because it has a condition handling `RetryNever`?